projectile.el

```diff
@@ -1779,6 +1780,11 @@ This is a subset of `grep-read-files', where either a matching entry from
                (and ext (concat "*." ext)))))
       (or default-alias default-extension))))
 
+(defun projectile-grep-ignored-patterns ()
```

Comment:
We already use the term patterns for things that came from `.projectile`, so I'd use some other terminology here.